helm/designate-certmanager-webhook/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "0.2.18"
 description: ACME webhook Implementation for OpenStack Designate
 name: designate-certmanager-webhook
-version: "0.3.1"
```

Comment:
In this PR you changed the structure of the value configuration, so this is at least a minor version increment, as it is sure a breaking change.

Reply:
I did the change as it is easy, rest LGTM